src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/X509Certificates/X509Certificate.cs

```diff
@@ -623,15 +576,19 @@ public virtual void Import(string fileName, SecureString? password, X509KeyStora
             throw new PlatformNotSupportedException(SR.NotSupported_ImmutableX509Certificate);
         }
 
-        internal ICertificatePalCore? Pal { get; private set; }
+        internal virtual ICertificatePalCore? Pal { get; private set; }
```

Comment:
Why make this virtual when there are no overrides?  Seems weird to turn a field access into a virtual method call.

Reply:
I was noodling with something else and didn't push the ref that undid that change. Sorry about that.

Reply:
(The noodling was "Use a covariant return type instead of shadowing on `X509Certificate2`")